ehitchhiking-ui/src/app/shared/components/menu/menu.component.ts

```diff
@@ -62,4 +65,44 @@ export class MenuComponent implements OnInit {
 	openProfileDialog(): void {
 		this.dialog.open(ProfileModalComponent, {panelClass: 'mat-dialog-no-padding', autoFocus: false});
 	}
+
+	private returnClass(type: NotificationTypes): string {
+		return NotificationComponent.headerClassMap[type];
+	}
+
+	callErrorNotification(notificationMessage: string) {
+		this.notificationBar.openFromComponent(NotificationComponent, {
+			duration: MenuComponent.notificationDuration,
+			data: {
+				message: notificationMessage,
+				type: NotificationTypes.error,
+			},
+			panelClass: [this.returnClass(NotificationTypes.error), 'notification-container'],
+			horizontalPosition: 'right',
+		});
+	}
+
+	callSuccessNotification(notificationMessage: string) {
+		this.notificationBar.openFromComponent(NotificationComponent, {
+			duration: MenuComponent.notificationDuration,
+			data: {
+				message: notificationMessage,
+				type: NotificationTypes.success,
+			},
+			panelClass: [this.returnClass(NotificationTypes.success), 'notification-container'],
+			horizontalPosition: 'right',
+		});
+	}
+
+	callInfoNotification(notificationMessage: string) {
+		this.notificationBar.openFromComponent(NotificationComponent, {
```

Comment:
pls create private function that will be generate
(message: string, type: enum) => {
			duration: MenuComponent.notificationDuration,
			data: {
				message,
				type,
			},
			panelClass: [this.returnClass(type), 'notification-container'],
			horizontalPosition: 'right',
		});
	}

Reply:
done